Automattic/jetpack
# Plans: Add new slugs for Backup and Security, deprecating the old ones

Fixes `1200412004370260-as-1200588199811619`.
Corresponds to https://github.com/Automattic/wp-calypso/pull/54222.

#### Changes proposed in this Pull Request:

* Add new plan/product slugs for the upcoming migration of all Backup and Security purchases to real-time:
  * `jetpack_backup(_monthly)`
  * `jetpack_backup_pro(_monthly)`
  * `jetpack_security(_monthly)`
  * `jetpack_security_pro(_monthly)`
* Include new slugs in the appropriate collections for monthly products, Backup products, Security products; in functions for converting among billing terms; etc.
* Update `JETPACK_FEATURE_PRODUCT_UPSELL_MAP` to reference new Backup and Security plans instead of the current (soon-to-be deprecated) ones.
* Keep existing slugs for Backup and Security plans, but mark references to them as deprecated, for easier removal later:
  * `jetpack_backup_daily(_monthly)`
  * `jetpack_backup_realtime(_monthly)`
  * `jetpack_security_daily(_monthly)`
  * `jetpack_security_realtime(_monthly)`

#### Jetpack product discussion

`p1HpG7-c8K-p2`
`pbtFFM-XN-p2`

#### Does this pull request change what data or activity we track or use?

Nope 🙂 

#### Testing instructions:

This PR doesn't have any feature implications, so there's nothing new to check out (yet). It could use some solid regression testing, though, to make sure no issues snuck in:

* Verify all the new slugs match perfectly with the ones added in Automattic/wp-calypso#54222.
* All tests should still pass (e.g., `pnpm test-adminpage`).
* All pages should still function as they currently do, with no changes. This PR should have no implications for the user experience.
* Specifically check that all product/plan icons on the At A Glance page display correctly.
